plugin.video.hrmagplayer/changelog.txt

```diff
@@ -0,0 +1,2 @@
+1.0.0
```

Comment:
In Krypton we have the `<news>` element in the addon.xml if you want to provide an easier way for users to check what is different between releases